src/main/java/com/minecolonies/core/colony/CitizenData.java

```diff
@@ -233,9 +234,9 @@ public class CitizenData implements ICitizenData
     protected final Map<Component, IInteractionResponseHandler> citizenChatOptions = new HashMap<>();
 
     /**
-     * If idle at job.
+     * The current status of the citizen's working (?)
      */
-    private boolean idle;
+    private JobStatus jobStatus;
```

Comment:
oh, we can just initialize this here with idle, so that it can't be null. simplifies reading then.